config/config.sample.yaml

```
@@ -1,17 +1,40 @@
-# Port for incoming slack requests from webhooks and event API messages.
-slack_hook_port: 9898
-# Prefix of incoming requests to strip when handing. This is NOT the bind host
-  #NOTE: Unlike most of the other urls, this one cannot use localhost,
-  #as this one must be publicly visible to the slack API
-inbound_uri_prefix: "https://my.server.here:9898/"
+homeserver: # Required
+  # The domain name of your homeserver
+  server_name: my.server.here
+  # The URL which the bridge server can reach the homeserver on.
+  # You can use localhost so long as the bridge and homeserver are
+  # hosted on the same machine
+  # 8008 is assumed to be the port the homeserver listens on
+  url: http://localhost:8008
+  # The public facing url for media on your homeserver.
+  # This is usually the public url of your homeserver.
+  # Optional, will use `url` by default.
```

```suggestion
  # Optional. The value of `url` will be used by default.
```

I initially read this as media_url will be set to `"url"`.